docs/data/material/components/alert/alert.md

```diff
@@ -8,83 +8,127 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/alert/
 
 # Alert
 
-<p class="description">An alert displays a short, important message in a way that attracts the user's attention without interrupting the user's task.</p>
+<p class="description">Alerts display brief messages for the user without interrupting their use of the app.</p>
 
-:::info
-This component is not documented in the [Material Design guidelines](https://m2.material.io/), but it is available in Material UI.
+## Introduction
+
+The Alert component can be used to provide important and potentially time-sensitive information in a way that does not interfere with the user's tasks. (Source: [ARIA APG](https://www.w3.org/WAI/ARIA/apg/patterns/alert/).)
+
```

Comment:
Maybe a very, very simple demo here? Like the one we recently did on the Accordion page.